persistence/modeshape-persistence-relational/src/main/resources/org/modeshape/persistence/relational/sqlserver_database.properties

```diff
@@ -30,4 +30,7 @@ update_content = UPDATE {0} SET LAST_CHANGED=CURRENT_TIMESTAMP, CONTENT = ? WHER
 remove_content = DELETE FROM {0} WHERE ID IN (#)
 
 # Remove all documents
-remove_all_content = DELETE FROM {0}
\ No newline at end of file
+remove_all_content = DELETE FROM {0}
+
+# Lock documents
+lock_content = SELECT ID FROM {0} WHERE ID IN (#) WITH (UPDLOCK) 
```

Comment:
ok, I can change this to UPLOCK and ROWLOCK. 

Note that the SQLServer support is really generic, since there's no way for me to actually test it :)